Core/include/Acts/Seeding/SeedFinderOrthogonal.ipp

```diff
@@ -680,6 +682,29 @@ void SeedFinderOrthogonal<external_spacepoint_t>::createSeeds(
    * seeing what happens if we take them to be our middle spacepoint.
    */
   for (const typename tree_t::pair_t &middle_p : tree) {
+    internal_sp_t &middle = *middle_p.second;
+    auto rM = middle.radius();
+
+    /*
+     * Cut: Ensure that the middle spacepoint lies within a valid r-region for
+     * middle points.
+     */
+    if (m_config.useVariableMiddleSPRange) {
+      if (rM < rMiddleMinSPRange) {
+        continue;
+      }
+      if (rM > rMiddleMaxSPRange) {
+        continue;
+      }
+    } else {
+      if (rM > m_config.rMaxMiddle) {
+        continue;
+      }
+      if (rM < m_config.rMinMiddle) {
+        continue;
+      }
```

Comment:
```suggestion
      if (rM < rMiddleMinSPRange || rM > rMiddleMaxSPRange) {
        continue;
      }
    } else {
      if (rM < m_config.rMinMiddle || rM > m_config.rMaxMiddle) {
        continue;
      }
```

Do you think it could easier to read like that?

Anyhow, I would suggest to switch the `if`s in the `else` to ask first for the min-condition and then the max-condition.

Reply:
Maybe you want also have a look into the Range functions in `Core/include/Acts/Utilities/Range1D.hpp` as @benjaminhuth just told me.

Reply:
Thanks for the comments. I changed the rMiddleMinSPRange to range1D in the default seeding recently but forgot to change it here 😵‍💫. Thanks for noticing it. 

Regarding the `if (rM < rMiddleMinSPRange || rM > rMiddleMaxSPRange) {` this might have to change in the future with the sorting of SPs but for now I agree it is easier to read.

Reply:
I changed 'rMiddleSPRange' to range1D to keep it consistent with the default seeding. But I think since there are several variables in the SeedFinder configuration that could be replaced by a range1D, including `rMinMiddle` and `rMaxMiddle` I will follow up this PR with this specific change